### ISSUE TITLE
When app turn from background to foreground, the console report an issue "UIApplication.endBackgroundTask(_:)"

### ISSUE THREAD

**Issue by StartingZhou** [firebase/firebase-ios-sdk#6937]
### When app Run in background, and turn it to foreground
The Xcode console will report an issue, but the app not crashed:
>2020-11-10 11:19:59.160400+0800 Noldor[3818:25962] [BackgroundTask] Background Task 83 ("GDTCCTUploader-upload"), was created over 30 seconds ago. In applications running in the background, this creates a risk of termination. Remember to call UIApplication.endBackgroundTask(_:) for your task in a timely manner to avoid this.

* Xcode version: Version 12.0 (12A7209)
  * Firebase SDK version: 
```
    pod 'Firebase/Core', '6.8.0'
    pod 'Firebase/Messaging', '6.8.0'
    pod 'Firebase/DynamicLinks', '6.8.0'
```
  * Installation method: `CocoaPods`
  
Is this issue will have a risk of crash? And why the GDTCCTUploader not call `UIApplication.endBackgroundTask(_:) ` when the app turn to foreground?

**Comment by morganchen12** [comment]
This is likely a duplicate of https://github.com/firebase/firebase-ios-sdk/issues/6258, which was fixed in version 6.33. Please try updating to the latest version and seeing if the issue is still present.

**Comment by StartingZhou** [comment]
> This is likely a duplicate of #6258, which was fixed in version 6.33. Please try updating to the latest version and seeing if the issue is still present.

Thank you!
